PyRDF/Proxy.py

```diff
@@ -25,6 +28,6 @@ def _call_handler(self, *args, **kwargs):
         if not self.action_node.value:
             generator = CallableGenerator(self.action_node._get_head())
             executor = Executor()
-            executor.execute(generator)
+            executor.execute(generator, Backend_env, Backend_conf)
```

Comment:
Actually, if these variables live in `Backend`, you will not even need to pass them here, you just pass the generator and `Backend` knows what to do with it.

Reply:
👍